packages/block-library/src/navigation-menu/edit.js

```diff
@@ -64,6 +64,8 @@ function NavigationMenu( {
 		} );
 	}, [ backgroundColor.class, textColor.class ] );
 
+	const emptyNavItemBlock = createBlock( 'core/navigation-menu-item' );
```

Comment:
I think it's best to invoke `createBlock( 'core/navigation-menu-item' )` inside `handleCreateEmpty`, otherwise we have an object as referenced by `emptyNavItemBlock` which can — later on, as this file evolves — mistakenly end up used in more than one place, leading to undetermined effects.

Reply:
Fixed.